### DISCUSSION TITLE
Help Request: [backend server not responding]

### DISCUSSION THREAD

**Discussion by rahulpro2001** [TEAMMATES/teammates#12682] [Help requests]
### Environment

after running command ./gradle runserver giving errors

### Intention

![Screenshot 2023-12-28 123205](https://github.com/TEAMMATES/teammates/assets/72226387/81a7aba0-7eb1-4d00-8dd7-c542a4572e20)

**Comment by cedricongjh** [comment]
hi @rahulpro2001 it appears that from the error output, a process is already using the port 8080, hence the dev server cannot bind to it, resulting in the error. I would suggest finding the process that is currently using the port and close it, then try running the command again
